### PR TITLE
Pr bebop2 gps

### PR DESCRIPTION
**Pull request by tumbili**
This enables gps on the bebop2. Amazing how fast that gps gets a fix, and I'm sitting on the couch in my house.

### CODE REVIEW

File: src/drivers/gps/gps.cpp [hunk 1]
```diff
@@ -510,6 +510,8 @@ int GPS::setBaudrate(unsigned baud)
 	int speed;
 
 	switch (baud) {
+	case 4800:   speed = B4800;   break;
```
Comment (tumbili) [inline src/drivers/gps/gps.cpp]: @LorenzMeier I didn't know which one it is so I added both, I guess it does no harm?

Comment (LorenzMeier) [inline src/drivers/gps/gps.cpp]: Now that you're saying it - it will slow down scanning. Can you please find out and remove the other one?

**Comment by LorenzMeier** [comment]
@tumbili I've fixed Bebop on master by reverting the offending commit. Can you rebase on master, push again here and test? It should all work now.

**Comment by tumbili** [comment]
@LorenzMeier Rebased on master and tested, all good. I will try to fly tomorrow, switching into position control works.

**Comment by LorenzMeier** [comment]
Thanks! Now circleCI has an issue. Code style or submodule?

**Comment by tumbili** [comment]
@LorenzMeier Forgot to pull the submodule from master. Should work now.

**Pull request merged**